src/frontend/Felles/HeaderMedSøk/utils.ts

```diff
@@ -124,11 +129,20 @@ export const lagÅpneEldreBehandlingerLenke = (): PopoverItem => {
     };
 };
 
-export const lagSamværskalkulatorLenke = (): PopoverItem => {
+export const lagSamværskalkulatorLenke = (personIdent: string | undefined): PopoverItem => {
+    if (!personIdent) {
+        return {
+            name: 'Samværskalkulator',
+            href: '/verktoy/samvaerskalkulator',
+            type: LenkeType.ARBEIDSVERKTØY,
+        };
+    }
     return {
         name: 'Samværskalkulator',
-        href: '/verktoy/samvaerskalkulator',
         type: LenkeType.ARBEIDSVERKTØY,
+        onSelect: async () => {
+            window.open(lagSamværskalkulatorLink(personIdent));
+        },
```

Comment:
```suggestion
        const urlSuffix = personIdent ? personIdent : '';

    return {
        name: 'Samværskalkulator',
        href: `/verktoy/samvaerskalkulator/${urlSuffix}`,
        type: LenkeType.ARBEIDSVERKTØY,
    };
```

I dette tilfellet går det fint å sløyfe bruken av onSelect